pyartcd/pyartcd/jenkins.py

```diff
@@ -317,13 +317,23 @@ def update_description(description: str, append: bool = True):
     set_build_description(build, description)
 
 
-def start_scan_osh(build_nvrs: list, email: Optional[str] = "", **kwargs):
-    params = {
-        'BUILD_NVRS': ','.join(build_nvrs)
-    }
+def start_scan_osh(build_nvrs: Optional[list] = None,
```

Comment:
This can be `image_nvrs` now

Reply:
Or we don't necessarily need to differentiate it here
container nvrs will always have `-container` and rpm won't
So we can just pass all nvrs to the job and the job then can sort them

Reply:
Hmm if I make the change now, it will break the scans that will be triggered by the `4.13` mass rebuild. Can I work on that as a refinement and raise the PR after the mass rebuild(s) are finished?

Reply:
Okay let's revisit this. For now this change is good